AsyncDisplayKit/Layout/ASStackLayoutChild.h

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Copyright (c) 2015-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+/**
+ Each child may override their parent stack's cross axis alignment.
+ @see ASStackLayoutAlignItems
+ */
+typedef NS_ENUM(NSUInteger, ASStackLayoutAlignSelf) {
```

Comment:
Most of the contents of this header was deleted, right?  Does it make sense to put it internal to the stack objects, or is there an unobtrusive header it could move to?  If we keep the separate file, let's name it ...Defines.h instead of ...Child.h.